### DISCUSSION TITLE
how can i prompt with a previous transcript to decrease errors

### DISCUSSION THREAD

**Discussion by ClaireCJS** [openai/whisper#1512] [Q&A]
I know whisper accepts prompts but I haven’t been able to successfully use them to my advantage.

I’m using whisper at the command line to create srt/lrc (same thing) files for music. These are a bit more than pure transcription because they return timing values. I don’t really know how that part works under the good, it’s all part of the whisper.exe and whisper-faster.exe distributions that i use.

Often times a TXT file of the lyrics to a song already exists.

When whisper makes the srt/lrc, it makes mistakes. Of course it’s not 100%.

Is there a way to prompt it with the TXT transcript so that it is less likely to make these mistakes?

I must be doing it wrong, because my prompt actually causes MORE mistakes.

TL;DR: I’m already using whisper to make SRT subtitle/karaoke files for music, but it makes mistakes, and I want to prompt it with pre-existing copies of the song’s lyrics so that it makes fewer mistakes.

I know it's kind of putting the horse before the cart, but in this case, some songs have TXT lyric transcripts already. I cannot produce a SRT/LRC from a TXT alone, the timing information requires an intelligence (be it artificial or natural). Whisper can do it, but it makes mistakes.

**Comment by phineas-pta** [comment]
that's not how prompts work with whisper

what u want is synchronizing / alignment of 2 transcripts

u can check out https://github.com/EtienneAb3d/WhisperTimeSync

**Reply by ClaireCJS** [reply]
omg thank you very much!!!!
